guides/rhai_expressions_cheat_sheet.cheatmd

```diff
@@ -0,0 +1,54 @@
+# Rhai expressions cheatsheet
+
+We are grouping into this cheatsheet the most frequent manipulations that can be done through Rhai, for convenience.
+
+## Cheatsheet
+{: .col-2}
+
+### Lists
+
+#### Filtering a list
+
+```rust
+[1, 2, 3].filter(|value| value % 2 == 0)
+=> [2]
+```
+
+#### Finding an element inside a list
+
+```rust
+repl> ["wow", "check"].index_of(|value| value == "check")
+=> 1
+```
+
+#### Checking if an expression is true for every element of a list
+
+```rust
+[2, 4, 6].all(|value| value % 2 == 0)
+=> true
+```
+
+### Maps
+
+#### Get only keys (in a list)
```

Comment:
This should probably be "Get only keys (in a map)" not "in a list"

Reply:
"in a list" means that it returns a list, maybe I should be more clear about that like `returns a list`